src/energyplus/ErrorFile.cpp

```diff
@@ -123,7 +126,10 @@ namespace energyplus {
             warningOrErrorString += "\n" + temp;
           }else{
             // unget the line
-            is.seekg(pos);
+            // is.seekg(pos);
+            // Instead of rewind then reread (which fails on Windows if you have LF line endings)
+            // We just use this bool to avoid having to re-read the line.
+            alreadyGotLine = true;
```

Comment:
If you have LF endings on Windows, after rewing you'd be missing the leading "*", eg
"* Warning** ...." instead of "** Warning **" and it woud fail.

Reply:
So instead of rewinding, just remember to NOT get the next line above on L105